config/puma.rb

```diff
@@ -1,20 +1,23 @@
 # Puma can serve each request in a thread from an internal thread pool.
-# The `threads` method setting takes two numbers a minimum and maximum.
+# The `threads` method setting takes two numbers: a minimum and maximum.
 # Any libraries that use thread pools should be configured to match
 # the maximum value specified for Puma. Default is set to 5 threads for minimum
-# and maximum, this matches the default thread size of Active Record.
+# and maximum; this matches the default thread size of Active Record.
 #
-threads_count = ENV.fetch("RAILS_MAX_THREADS") { 5 }.to_i
+threads_count = ENV.fetch("RAILS_MAX_THREADS") { 5 }
```

Comment:
[pronto-rubocop warning] Style/RedundantFetchBlock: Use `fetch("RAILS_MAX_THREADS", 5)` instead of `fetch("RAILS_MAX_THREADS") { 5 }`.